### ISSUE TITLE
Sample

### ISSUE THREAD

**Issue by andrewlord1990** [andrewlord1990/unity-socialplay#4]

**Comment by keiranlovett** [comment]
Interested in using package, not sure how to start - a sample with basic implementation would be great. For example, will I need to include anything in the scene? @andrewlord1990

**Comment by andrewlord1990** [comment]
Excellent. So far just written to use myself, been meaning to create a a
sample using it. Will try and put something together soon.

**Comment by keiranlovett** [comment]
Is it in a usable state currently by the way?

**Comment by andrewlord1990** [comment]
I use it currently in a game. However, I haven't tested it enough to call
it production-ready yet. As you can see on the issues page i have some
stuff to do on it. The code is based on something i have it a live-game,
but i did make some tweaks to it that haven't yet been tested on
android/iOS yet.

On 2 Apr 2017 11:45, "Keiran" <notifications@github.com> wrote:

Is it in a usable state currently by the way?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/andrewlord1990/unity-socialplay/issues/4#issuecomment-290978394>,
or mute the thread

**Comment by keiranlovett** [comment]
Mind if I roll into a game in production now? I'll roll any issues / bug fixes through pushes to this then?

**Comment by andrewlord1990** [comment]
Yeah sure, of course. Just wanted to be clear, so you knew where i was with
it.

**Comment by keiranlovett** [comment]
Continuing comments in this issue as I don't believe its value in opening another issue. I have been able to get working on Android, but iOS is not working. Is there any dependencies?

**Comment by andrewlord1990** [comment]
Ah okay. No, Game Center should work out of the box.

I have cleaned up the code a bit and added code documentation comments.
However, testing on iOS is a bit of a pain as you need a game already
uploaded to iTunesConnect I believe.

**Comment by keiranlovett** [comment]
That's what I thought glancing through the code, and wanted to check - Turns out Game Center was disabled on my debug phone, and I've just resolved. 

The documentation is stellar!

**Comment by andrewlord1990** [comment]
I have altered how the classes are structured, made the main class a MonoBehaviour, added code documentation and a Prefab to use. This is all on master branch, you can build it straight from there with the export-package script. (Details in the .travis script.)

Once I have added the sample project I will release a new version on GitHub releases.

**Comment by andrewlord1990** [comment]
@keiranlovett I have written a sample scene. Will release a new version which contains it, once the PR is merged.
